Quickstart/azure-search-quickstart/azure-search-quickstart/Program.cs

```diff
@@ -0,0 +1,267 @@
+﻿namespace azure_search_quickstart
```

Comment:
Mentioned this in the docs: The underscore_convention is typically avoided in .NET in favor of PascalCase.

Reply:
Done